q2_feature_classifier/_taxonomic_classifier.py

```diff
@@ -69,7 +69,17 @@ def _1(dirfmt: TaxonomicClassiferTemporaryPickleDirFmt) -> Pipeline:
         tmpdir = model.DirectoryFormat()
         dirname = str(tmpdir)
         tar.extractall(dirname)
-        pipeline = joblib.load(os.path.join(dirname, 'sklearn_pipeline.pkl'))
+        try:
+            pipeline = joblib.load(os.path.join(dirname,
+                                                'sklearn_pipeline.pkl'))
+        except MemoryError:
+            raise MemoryError("There is a resource issue regarding your "
```

Comment:
How about we replace the first sentence with something like what is in the [python stdlib docs](https://docs.python.org/3/library/exceptions.html#MemoryError):

> The operation has run out of available memory. To correct this error:...

Reply:
I changed the error message as requested.